sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -213,6 +214,17 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
           userTypes: [InstitutionUserTypes.admin],
         },
       },
+      {
+        path: AppRoutes.InstitutionUserProfile,
+        name: InstitutionRoutesConst.INSTITUTION_USER_PROFILE,
+        components: {
+          default: InstitutionUserProfile,
+          sidebar: InstitutionHomeSideBar,
+        },
+        meta: {
+          clientType: ClientIdType.INSTITUTION,
```

Comment:
I am not sure if we need explicitly set the user types also. @ann-aot, can you please confirm it? It would be something as below:
```
userTypes: [
            InstitutionUserTypes.admin,
            InstitutionUserTypes.locationManager,
            InstitutionUserTypes.user,
          ],
```

Reply:
sorry @dheepak-aot  I mean, for institutions you will need to add the userType. as @andrewsignori-aot  said it is working for you since the student router does any restriction, we need to adding logic for the student router in future sprint